apps/angular/ionic-angular-standalone/src/app/issue/lifecycle/lifecycle.page.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { IonicModule, ViewWillEnter, ViewDidEnter, ViewWillLeave, ViewDidLeave } from '@ionic/angular';
+
+@Component({
+  selector: 'app-lifecycle',
+  templateUrl: './lifecycle.page.html',
+  styleUrls: ['./lifecycle.page.scss'],
+  standalone: true,
+  imports: [IonicModule, CommonModule, FormsModule]
+})
+/**
+ * Error due to `@ionic/angular/standalone` not containing a lifecycle.
+ */
+export class LifecyclePage implements OnInit, ViewWillEnter, ViewDidEnter, ViewWillLeave, ViewDidLeave {
```

Comment:
Error due to `@ionic/angular/standalone` not containing a lifecycle.

Reply:
This looks to be a missing set of exports from Ionic Framework. This will be addressed here: https://github.com/ionic-team/ionic-framework/pull/28346!